### PR TITLE
Analytics timer

### PR DESCRIPTION
**Pull request by phoebemirman**
add 30s timer to send analytic tracking events

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://dashboard.featurepeek.com). 🌻
Your deployment will be kept up-to-date with this pull request's latest changes.

**https://peek.run/1rehow3**

-------

*Please read [our docs](https://docs.featurepeek.com/configuring-peek-yml/) for more configuration details.*

**Comment by bthaile** [comment]
seems there is a less code-intensive way to delay by 30 seconds. 
I don't think we need to use redux for analytics. I rather just put a queue in front of mech that sends the analytics out. Then send using a setTimeout(I)=> ..., 30 * 60 *) and also detect if user is going to either nav away or close tab then send all analytics. 

then we won't need redux.

File: packages/augur-ui/src/reducers/index.ts [hunk 1]
```diff
@@ -53,8 +54,10 @@ import {
   Drafts,
   MarketsList,
   ReportingListState,
+  Analytics
 } from "modules/types";
 import { Getters } from "@augurproject/sdk";
+import { AnnotationsTunnelTypeHeightControlPointOptions } from "highcharts";
```
Comment (bthaile) [inline packages/augur-ui/src/reducers/index.ts]: did this get added by accident?

Comment (phoebemirman) [reply packages/augur-ui/src/reducers/index.ts]: oops yea

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://dashboard.featurepeek.com). 🥳
Your deployment will be kept up-to-date with this pull request's latest changes.

**https://peek.run/kq3h547**

-------

*Please read [our docs](https://docs.featurepeek.com/configuring-peek-yml/) for more configuration details.*

**Review by bthaile — APPROVED** [review]
lgtm

**Pull request merged**